src/main/java/com/exasol/adapter/dynamodb/queryplanning/RemoteTableQuery.java

```diff
@@ -15,7 +19,8 @@
     private static final long serialVersionUID = -721150997859918517L;
```

Comment:
Should serialuid also change?

Reply:
Maybe yes. It's not so important here as serialization is only used during query execution.